contract/tests/test_contract.py

```diff
@@ -12,17 +12,28 @@
 
 from odoo import fields
 from odoo.exceptions import UserError, ValidationError
-from odoo.tests import Form, common
+from odoo.tests import Form, common, tagged
 
 
 def to_date(date):
     return fields.Date.to_date(date)
 
 
+@tagged("post_install", "-at_install")
 class TestContractBase(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        if not cls.env.company.chart_template_id:
+            # Load a CoA if there's none in current company
+            coa = cls.env.ref("l10n_generic_coa.configurable_chart_template", False)
+            if not coa:
+                # Load the first available CoA
+                coa = cls.env["account.chart.template"].search(
```

Comment:
Moreover, IMHO, this is not correct. If several coa's have been installed, how can you be sure the good one will be loaded for the current company ?

Reply:
IMHO, this kind of code should be set in particular modules, not all.